.env.example

```diff
@@ -1 +1,4 @@
 OPENAI_API_KEY=REPLACE
+
+# For Experiment 02-assistant
+ASSISTANT_ID=REPLACE
```

Comment:
> This is more of an identifier than a secret, yeah? 

Exactly. Just config. I wasn't sure if there is any way it could be abused, and didn't want to check into the repo without being more confident (I would imagine its likely fine and doesn't work without our API key). 

>or does it make more sense leave this blank to allow devs to iterate in parallel without conflicts?

Yeah, this was also part of the reasoning. 

Happy to remove if we would like to keep `.env` purely for secrets. Not strongly opinionated here.

Reply:
Makes sense this way